pkg/microservice/aslan/core/common/repository/mongodb/test_task_stat.go

```diff
@@ -90,9 +90,9 @@ func (c *TestTaskStatColl) ListTestTask() ([]*models.TestTaskStat, error) {
 
 func (c *TestTaskStatColl) FindTestTaskStat(option *TestTaskStatOption) (*models.TestTaskStat, error) {
 	query := bson.M{"name": option.Name}
-	testTaskStat := new(models.TestTaskStat)
+	var testTaskStat *models.TestTaskStat
 
-	err := c.FindOne(context.TODO(), query).Decode(testTaskStat)
+	err := c.FindOne(context.TODO(), query).Decode(&testTaskStat)
```

Comment:
`&` is not necessary here

Reply:
It is necessary, otherwise the data will not be available